firstvoices/backend/models/characters.py

```diff
@@ -239,21 +240,23 @@ class Meta:
     # JSON representation of a g2p mapping from confusable characters to canonical characters
     input_to_canonical_map = models.JSONField(default=list)
 
-    @property
+    @cached_property
```

Comment:
So these 3 properties were being used in multiple places, and since all these properties are DB calls, we were getting multiple similar queries for the 3 related models - `Character, CharacterVariant and IgnoredCharacter`.
Using a `@cached_property` reduces those duplicate queries as the properties only get evaluated once for an instance.

The only catch is that we should not be modifying any of these 3 related models within the same request/response cycle 
or
Any other place where we are using an `Alphabet` instance and we modify any of the 3 models, and then want to use the updated results. In that case, we will have to manually refresh the property.

I did glance through the code where we are using `Alphabet` model, and also verified with Bronson if it is safe to cache these properties. All the tests are also passing. 
But please inform if there are any edge cases where we can run into problem.
Thanks :)